cgi/manifest.pl

```diff
@@ -65,6 +65,7 @@
 $manifest{start_url} = format_subdomain($subdomain);
 $manifest{scope} = '/';
 $manifest{display} = 'standalone';
+$manifest{prefer_related_applications} = true;
```

Comment:
This needs to be 'true' instead of true to make it a string, there is no boolean type in Perl.

Reply:
I think it should probably be `$JSON::PP::true`.